kernelcare/datadog_checks/kernelcare/check.py

```diff
@@ -0,0 +1,71 @@
+import requests
+
+from datadog_checks.base import AgentCheck, ConfigurationError
+from datadog_checks.base.errors import CheckException
+
+
+class KernelcareCheck(AgentCheck):
+
+    KEY_KCARE_NAGIOS_ENDPOINT = 'https://cln.cloudlinux.com/clweb/api/kcare/nagios/'
+    RES_KCARE_NAGIOS_ENDPOINT = 'https://cln.cloudlinux.com/clweb/api/kcare/nagios-res/'
+
+    HTTP_RESPONSE_ERRORS = [
+        'Servers not found for key',
+        'Reseller not found',
+        'Registration token not found for reseller',
+    ]
+
+    def _parse_nagios_response(self, text):
+
+        lines = text.split('\n')
+        tmp = lines[0].split('|', 1)
+        if len(tmp) == 1:
+            params = tmp[0]
+        else:
+            params = tmp[1]
+
+        res = {}
+        for p in params.split(';'):
+            k, v = p.split('=', 1)
+            res[k] = int(v)
+        return res
+
+    def get_url(self, instance):
+
+        key = instance.get('key')
+
+        if key:
+            return self.KEY_KCARE_NAGIOS_ENDPOINT + key
+
+        login = instance.get('login')
+        api_token = instance.get('api_token')
+
+        if login and api_token:
+            return self.RES_KCARE_NAGIOS_ENDPOINT + login + '/' + api_token
+
+        raise ConfigurationError('Configuration error, you must provide `key` or `login` & `api_token`')
+
+    def check(self, instance):
+
+        url = self.get_url(instance)
+
+        response = requests.get(url)
```

Comment:
Instead of using `requests.get`, could you use the http wrapper? The wrapper allows the integration to support all of our standardized configuration options (proxy, tls validation, etc) out of the box. https://datadoghq.dev/integrations-core/base/http/

Reply:
switched to http wrapper